File: airbyte-integrations/connectors/destination-mysql/src/test-integration/java/io/airbyte/integrations/destination/mysql/SshKeyMySQLDestinationAcceptanceTest.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.destination.mysql;

import java.nio.file.Path;
import org.junit.jupiter.api.Disabled;

@Disabled
public class SshKeyMySQLDestinationAcceptanceTest extends SshMySQLDestinationAcceptanceTest {

  @Override
  public Path getConfigFilePath() {
    return Path.of("secrets/ssh-key-config.json");
  }

}

File: airbyte-integrations/connectors/destination-mysql/src/test-integration/kotlin/io/airbyte/integrations/destination/mysql/SshKeyMySQLDestinationAcceptanceTest.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.integrations.destination.mysql

import java.nio.file.Path
import org.junit.jupiter.api.Disabled

@Disabled
class SshKeyMySQLDestinationAcceptanceTest : SshMySQLDestinationAcceptanceTest() {
    override val configFilePath: Path?
        get() = Path.of("secrets/ssh-key-config.json")
}
